### PR TITLE
Enhance the github shim for future v1 to v3 rbac changes.

### PR DESCRIPTION
**Pull request by jctanner**
separated out of https://github.com/ansible/galaxy_ng/pull/1853 to make smaller PRs

### CODE REVIEW

**Review by rochacbruno — APPROVED** [review]

File: profiles/community/github_mock/flaskapp.py [hunk 1]
```diff
@@ -48,22 +52,26 @@
...
-        'password': 'redhat'
+        'password': 'redhat',
+        'email': 'jctannerTEST@gmail.com',
     },
     'geerlingguy': {
         'id': 1004,
         'login': 'geerlingguy',
-        'password': 'redhat'
+        'password': 'redhat',
+        'email': 'geerlingguy@gmail.com',
```
Comment (rochacbruno) [inline profiles/community/github_mock/flaskapp.py:74]: Is it okay to use a real e-mail address here?

Comment (jctanner) [reply profiles/community/github_mock/flaskapp.py:74]: I made it up ... I guess it could possibly be his real email?

Comment (rochacbruno) [reply profiles/community/github_mock/flaskapp.py:74]: @jctanner it is his real e-mail

**Pull request merged**